system_baseline/models.py

```diff
@@ -59,18 +59,23 @@ def mapped_system_ids(self, rbac_group_filters=None):
         # [{'id': None}, {'id': '39eb2f52-37c1-4f75-95e6-7237d0b385b7'}]
 
         mapped_systems_query = self.mapped_systems
+
+        def _get_query_from_filters(filters):
+            return [
+                SystemBaselineMappedSystem.groups == "[]"
+                if "id" in rbac_group_filter and rbac_group_filter["id"] is None
+                else SystemBaselineMappedSystem.groups.contains(cast([rbac_group_filter], JSONB))
+                for rbac_group_filter in filters
```

Comment:
`rbac_group_filter` looks similar to `rbac_group_filters` from outer function.

Reply:
This is the remnant of refactoring and moving the code. Thank you!